net-im/signal-cli-bin/files/signal-cli-bin-0.7.4-use-working-java-version.patch

```diff
@@ -0,0 +1,31 @@
+diff -Naur a/bin/signal-cli b/bin/signal-cli
+--- a/bin/signal-cli	2020-10-14 18:39:40.000000000 +0200
++++ b/bin/signal-cli	2020-11-04 09:20:25.397469784 +0100
+@@ -84,13 +84,20 @@
+ 
+ 
+ # Determine the Java command to use to start the JVM.
+-if [ -n "$JAVA_HOME" ] ; then
+-    if [ -x "$JAVA_HOME/jre/sh/java" ] ; then
+-        # IBM's JDK on AIX uses strange locations for the executables
+-        JAVACMD="$JAVA_HOME/jre/sh/java"
+-    else
+-        JAVACMD="$JAVA_HOME/bin/java"
+-    fi
++# This package needs Java 11, but Java 11 is currently not part of the gentoo eselect config system
++# so we must search for matching java binaries
++JAVA_DIRS="$JAVA_HOME /opt/openjdk-jre-bin-11 /opt/openjdk-bin-11"
```

Comment:
it should also check /usr/lib*/openjdk-11 first.

I suggest getting rid of JAVA_DIRS and do iteration literally, in that order:
`for I in  /usr/lib*/openjdk-11 /opt/openjdk-bin-11 /opt/openjdk-jre-bin-11; do`

Reply:
note that * has to be outside of the quotes to work. it will handle both lib and lib64 cases that way.

Reply:
Hello @gyakovlev , 
good hint!
I will prepare a PR for it.

Reply:
@gyakovlev please see https://github.com/gentoo/gentoo/pull/21958